wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -15,6 +15,174 @@
 
 <main id="main" class="site-main">
 
+<?php
+if ( '' === get_query_var( 'search' ) && empty( $_GET ) && is_post_type_archive() ) :
+	?>
+	<section>
+		<div class="row align-middle between section-heading section-heading--with-space">
+			<h1 class="section-heading_title h2"><?php esc_html_e( 'Lesson Plans', 'wporg-learn' );?></h1>
+			<a href="<?php echo esc_url( get_post_type_archive_link( 'lesson-plan' ) . '?_view=all' ); ?>" class="button button-xlarge button-secondary">
+				<?php esc_html_e( 'Browse all lesson plans', 'wporg-learn' ); ?>
+			</a>
+		</div>
+
+		<hr>
+
+		<div class="row align-middle between section-lp_description">
+			<p class="section-lp_text">
+				<?php esc_html_e( 'Want to help others learn about WordPress? Read through, use, and remix these lesson plans.', 'wporg-learn' );?>
+			</p>
+			<?php
+			set_query_var( 'post_type', 'lesson-plan' );
+			get_template_part( 'template-parts/component', 'archive-search' );
+			?>
+		</div>
+
+		<hr>
+		<?php
+		$categories = get_terms( 'wporg_lesson_category', array(
+			'hide_empty' => false,
+			'orderby'    => 'id',
+		) );
+		?>
+		<div class="row lp-taxonomy categories">
+			<div class="card-grid card-grid_4">
+				<h2 class="h4 lp-taxonomy-header"><?php echo esc_html__( 'Topic', 'wporg-learn' ); ?></h2>
+				<div class="lp-taxonomy-description"><?php echo esc_html__( 'Browse lesson plans by their high-level topic.', 'wporg-learn' ); ?></div>
+				<?php foreach ( $categories as $category ) :
+					$is_sticky = get_term_meta( $category->term_id, 'sticky', true );
+					if ( $is_sticky ) :
+						?>
+				<div class="card">
+					<a href="<?php echo esc_url( get_term_link( $category ) ); ?>">
+						<?php
+						$category_icon = get_term_meta( $category->term_id, 'dashicon-class', true ) ? get_term_meta( $category->term_id, 'dashicon-class', true ) : 'wordpress-alt';
```

Comment:
```suggestion
						$category_icon = get_term_meta( $category->term_id, 'dashicon-class', true ) ?? 'wordpress-alt';
```

This should be equivalent, but easier to read, and doesn't execute the function twice.

https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op